R/get.version.R

```diff
@@ -23,16 +23,15 @@ get.version <- function(pkg, date, current.deps = c("Rcpp")) {
   if (dfk$Published[1] > date) {
     message2()
     message1("According to our records, the package: '", pkg, "' was not yet available on CRAN on '", date, "'")
-    stop()
+    stop2()
   }
   # 2.3 Check if date requested comes before most up to date date
   last.toc.date <- max(cran.toc$Published, na.rm = TRUE)
   if (date > last.toc.date) {
-    stop(
+    stop2(paste0(
```

Comment:
```suggestion
    stop(
```

This is a legit error